src/multibody/geometry.hpp

```diff
@@ -274,100 +268,61 @@ struct GeometryObject
 
     GeometryModel(const se3::Model & model)
       : model(model)
-      , ncollisions(0)
-      , nvisuals(0)
-      , collision_objects()
-      , visual_objects()
+      , ngeoms(0)
+      , geometry_objects()
```

Comment:
Can't we find a better name for geometry_object? 
- fclObject  : I still dont like "object", but with "fcl" attached should be clear enough
- fclCollisionObject : object is not adding meaning, collision is confusing wrt visual
- fclWrapper: better than fclObject, but still not very clear
- fclGeometry: in fcl, they use "geometry" for the 3D volume, i.e. something contain in their CollisionObject.
- fclGeometryWrapper: quite long.